src/desktop/apps/partner2/client/overview_layout_factory.coffee

```diff
@@ -55,6 +55,15 @@ module.exports = (partner, profile) ->
       galleryThreeArtists partner, profile
       galleryThreeArticles partner, profile
     ]
+    gallery_four: [
+      galleryThreeHero partner, profile
+      galleryTwoAbout partner, profile
+      galleryTwoNews partner, profile
+      galleryTwoShows partner, profile
+      galleryTwoFairBooths partner, profile
+      galleryTwoArtists partner, profile
+      galleryTwoArticles partner, profile
+    ]
```

Comment:
If all the `galleryTwo___` is equal to `galleryThree___`, then wouldn't this mean that `galllery_four` is equal to `gallery_three`? Maybe it's better to have a separation in case there are other changes to the new subscription feature sets.

Reply:
That's a really good point. I could simplify this file by reducing all of those variables to just the original section, but I think the correct thing to do is make `galleryFour____` consistent with `galleryThree___` and `galleryTwo___` by explicitly create renamed copies of each section.